### DISCUSSION TITLE
Root - oh-my-posh: command not found

### DISCUSSION THREAD

**Discussion by o0Noise0o** [JanDeDobbeleer/oh-my-posh#5370] [Troubleshoot] [answered]
HI all, Oh my posh is working perfect in normal user mode.
But when i switch to root it stops working with the following error:  oh-my-posh: command not found
and then shows an normal bash cl. Can u tell me what to do?

i am using ~./profile

**Comment by JanDeDobbeleer** [comment, marked as answer]
@o0Noise0o the location of oh-my-posh isn't added to your PATH for the root user (done manually in your rc file).

**Reply by o0Noise0o** [reply]
Thank u, another lesson learned.
